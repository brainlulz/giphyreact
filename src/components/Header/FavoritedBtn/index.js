import FavoritedBtn from './FavoritedBtn';
export default FavoritedBtn;