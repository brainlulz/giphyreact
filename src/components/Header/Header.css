.Header {
    grid-column: 2/3;
    grid-row: 1/1;
    display: grid;
    grid-template-columns: 1fr;
    grid-auto-rows: minmax(3rem, auto);
    align-items: center;
}