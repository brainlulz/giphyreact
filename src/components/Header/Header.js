import React from "react";
import Title from "./Title";
import FavoritedBtn from "./FavoritedBtn";

import "./Header.css";

const header = ({ favClicked }) => {
  return (
    <div className="Header">      
      <Title />
      <FavoritedBtn favClicked={favClicked}/>
    </div>
  );
};

export default header;
