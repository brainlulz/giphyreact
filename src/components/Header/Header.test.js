import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('Header', () => {
  it('renders the Header component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
