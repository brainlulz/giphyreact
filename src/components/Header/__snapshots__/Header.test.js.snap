// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders the Header component 1`] = `
<div
  className="Header"
>
  <Title />
  <Favorited />
</div>
`;
